src/main/scala/org/spartanz/parserz/ParsersModule.scala

```diff
@@ -21,7 +21,30 @@ trait ParsersModule {
       Map[SI, SO, E1, A, B](self, asEither(e)(to.lift), asEither(e)(from.lift))
```

Comment:
Everywhere in this file, I'd make sure you are aggressively lazy, so we can have recursive grammars. It will require a trick (identity map) to derive the graph from that, but it can be done.

Reply:
I have implemented recursive grammars with the explicit `Delay` and corresponding combinator, please see

https://github.com/spartanz/parserz/blob/c2d42d223c777391767e32c4d643a32d28f584f4/src/main/scala/org/spartanz/parserz/ParsersModule.scala#L84-L85

I'll try to allow recursion out of the box, without additional work at the call site. Thanks!

Reply:
#53